src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2520,7 +2524,8 @@ static bool appendContextAttributeItem
 
   /* Check aspects related with location */
   if (!processLocationAtAppendAttribute(currentLocAttrName, targetAttr, actualAppend, geoJson,
-                                        &err, apiVersion, oe))
+                                        &err, apiVersion, oe)
+      || !processDateExpirationAtAppendAttribute(dateExpiration, targetAttr, actualAppend, &err, oe))
```

Comment:
Same as previous comment https://github.com/telefonicaid/fiware-orion/pull/3173/files#r187559469

Reply:
NTC by the moment (I'll re-review on the new version of the code, rasing a comment if needed)